zap.go

```diff
@@ -0,0 +1,64 @@
+package golog
+
+import (
+	"errors"
+	"fmt"
+	"log"
+
+	"go.uber.org/zap"
+)
+
+type zapper struct {
```

Comment:
You could just redefine zap.SugaredLogger like this:

`type zapper zap.SugaredLogger`

Reply:
Nice